src/hooks/proxy.ts

```
@@ -0,0 +1,28 @@
+import { HttpsProxyAgent } from 'https-proxy-agent';
+import { URL } from 'url';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)